src/static/js/modules/util/utilities.js

```diff
@@ -4,6 +4,8 @@
 
 'use strict';
 
+var typeCheckers = require( './utilities-type-checkers' );
```

Comment:
The name could possibly be shortened to `type-checkers.js` since it's in the `util` directory, but either works. We don't have this happening currently consistently, but I believe a best practice is that the name of the module reference follow its filename (but as camelCase), e.g. `var utilitiesTypeCheckers = require( './utilities-type-checkers' );` or `var typeCheckers = require( './type-checkers' );`.